src/frontends/pytorch/src/op/squeeze.cpp

```diff
@@ -0,0 +1,25 @@
+
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_squeeze(NodeContext& context) {
+    auto inputs = context.inputs();
```

Comment:
Please validate that there is at least 1 input in `inputs`

Reply:
added checks